proposals/0272-sdl-javascript-manager-layer.md

```diff
@@ -625,13 +625,13 @@ class ScreenManagerBase extends SubManagerBase {
     setDynamicMenuUpdatesMode (value)
 
     /**
-     * @return {DynamicMenuUpdatesMode} - The currently set DynamicMenuUpdatesMode. It defaults to ON_WITH_COMPAT_MODE if not set.
+     * @return {DynamicMenuUpdatesMode} - The currently set DynamicMenuUpdatesMode. It defaults to ON_WITH_COMPAT_MODE if not set
```

Comment:
```suggestion
     * @return {DynamicMenuUpdatesMode} - The currently set DynamicMenuUpdatesMode. It defaults to ON_WITH_COMPAT_MODE if not set.
```
I'd keep the period here, since you use it in the sentence before.